src/renderer/screens/Preferences/MyKeyboard.js

```diff
@@ -301,59 +184,131 @@ const AdvancedKeyboardSettings = () => {
+        <FormControl variant="standard" fullWidth={true}>
+          <InputLabel>{t("keyboardSettings.keymap.defaultLayer")} </InputLabel>
+          <Select
+            onChange={selectDefaultLayer}
+            sx={{ mb: 2 }}
+            value={defaultLayer}
+            variant="filled"
+            input={<FilledInput sx={{}} />}
+          >
+            <MenuItem value={126}>
+              {t("keyboardSettings.keymap.noDefault")}
+            </MenuItem>
+            {layers}
+          </Select>
+        </FormControl>
+
+        {ledIdleTimeLimit >= 0 && (
+          <FormControl variant="standard" fullWidth={true}>
+            <InputLabel>{t("keyboardSettings.led.idleTimeLimit")}</InputLabel>
+            <Select
+              onChange={selectIdleLEDTime}
+              value={ledIdleTimeLimit}
+              variant="filled"
+              input={<FilledInput sx={{}} />}
+            >
+              <MenuItem value={0}>
+                {t("keyboardSettings.led.idleDisabled")}
+              </MenuItem>
+              <MenuItem value={60}>
+                {t("keyboardSettings.led.idle.oneMinute")}
+              </MenuItem>
+              <MenuItem value={120}>
+                {t("keyboardSettings.led.idle.twoMinutes")}
+              </MenuItem>
+              <MenuItem value={180}>
+                {t("keyboardSettings.led.idle.threeMinutes")}
+              </MenuItem>
+              <MenuItem value={240}>
+                {t("keyboardSettings.led.idle.fourMinutes")}
+              </MenuItem>
+              <MenuItem value={300}>
+                {t("keyboardSettings.led.idle.fiveMinutes")}
+              </MenuItem>
+              <MenuItem value={600}>
+                {t("keyboardSettings.led.idle.tenMinutes")}
+              </MenuItem>
+              <MenuItem value={900}>
+                {t("keyboardSettings.led.idle.fifteenMinutes")}
+              </MenuItem>
+              <MenuItem value={1200}>
+                {t("keyboardSettings.led.idle.twentyMinutes")}
+              </MenuItem>
+              <MenuItem value={1800}>
+                {t("keyboardSettings.led.idle.thirtyMinutes")}
+              </MenuItem>
+              <MenuItem value={3600}>
+                {t("keyboardSettings.led.idle.sixtyMinutes")}
+              </MenuItem>
+            </Select>
+          </FormControl>
+        )}
+        {ledBrightness >= 0 && (
+          <FormControl variant="standard" fullWidth={true}>
+            <Typography variant="caption" gutterBottom>
+              {t("keyboardSettings.led.brightness")}
+            </Typography>
+            <Slider
+              max={255}
+              value={ledBrightness}
+              sx={{ width: "300" }}
+              onChange={setBrightness}
+            />
+          </FormControl>
+        )}
+      </FormGroup>
+      <Box sx={{ my: 1, display: "flex" }}>
+        <Box sx={{ flexGrow: 1 }} />
+        <Button onClick={saveKeymapChanges} disabled={!modified}>
```

Comment:
In the screenshot, this looks funny when disabled.  do we need a style on it? 

I'd probably just stick with the shared savechangesbutton so we can ensure consistent styling

Reply:
Can do. I wanted to move a bit away from `SaveChangesButton`, because with partial transparency, it doesn't look all that good, and we don't really need a progress indicator here, because these the settings here save fast enough to almost be faster than rendering any noticable amount of animation.

So I'd rather use a normal button here. Perhaps with an outline, to make it more buttonish?